node/bft-consensus/tests/basics.rs

```diff
@@ -99,12 +99,11 @@ async fn verify_state_coherence() {
 #[tokio::test(flavor = "multi_thread")]
 async fn primary_failures() {
     // Configure the primary-related variables.
-    const NUM_PRIMARIES: usize = 4;
-    const WORKERS_PER_PRIMARY: u32 = 1;
+    const NUM_PRIMARIES: usize = 4; // this shouldn't be altered right now
     const PRIMARY_STAKE: u64 = 1;
 
     // Configure the transactions.
-    const NUM_TRANSACTIONS: usize = 30;
+    const NUM_TRANSACTIONS: usize = 30; // this shouldn't be altered right now either
```

Comment:
Maybe this comment could be made more useful if it explains why and when this should be changed? 
Ditto with the number of primaries.